be/src/runtime/runtime_state.h

```diff
@@ -306,6 +306,8 @@ class RuntimeState {
     void set_enable_pipeline_engine(bool enable_pipeline_engine) { _enable_pipeline_engine = enable_pipeline_engine; }
     bool enable_pipeline_engine() const { return _enable_pipeline_engine; }
 
+    bool use_sorted_agg() const { return _use_sorted_agg; };
```

Comment:
`RuntimeState` is for one fragment? What if there are multiple aggregations in a fragment? Should this option be at the aggregation node level rather than fragment level?

Reply:
RuntimeState::use_sorted_agg() will never used anyone. I will delete it.